react/week5/github-search/src/routercomponent.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import Context from "./context";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import { useParams, Link } from "react-router-dom";
+import Card from "react-bootstrap/Card";
+import Column from "react-bootstrap/Col";
+
+function UserRouter() {
+  const [followers, setFollowers] = React.useState([]);
+  const { Todo } = React.useContext(Context);
+  const { userId } = useParams();
+  const mainuser = Todo.find((users) => users.id === Number(userId));
+  //===================================================================For followers information */
+
+  const URL = `https://api.github.com/users/${mainuser.login}/followers`;
+  React.useEffect(() => {
+    fetch(URL)
+      .then((response) => response.json())
+      .then((data) => setFollowers(data));
+  }, [URL]);
+  return (
+    //=================================================================== Main user container */
```

Comment:
These types of comments generally aren't necessary. The declarative nature of React is usually enough for people to understand your code.